### DISCUSSION TITLE
Can you change user properties via CLI?

### DISCUSSION THREAD

**Discussion by scraejtp** [goauthentik/authentik#7562] [Q&A] [answered]
I found myself locked out of Authentik due to an accidental mismanagement of the the user property is_active.

While I can create a recovery flow to login via https://goauthentik.io/docs/troubleshooting/login, it appears not to function as the akadmin account has the property is_active set to false.

Any way to correct this beyond stepping back to a backup?

**Comment by scraejtp** [comment, marked as answer]
Found an answer to my problem in the docs.

https://goauthentik.io/docs/troubleshooting/missing_admin_group

**Reply by faustlod** [reply]
How were you able to use the token? For me, it only generates a /recovery/use-token/#verylongtoken path.

**Reply by scraejtp** [reply]
The first link for troubleshooting login? It has been a while but that link Authentik generates should allow you to login bypassing the normal flow you have designated.

**Reply by faustlod** [reply]
In CLI, I execute the following command: ak create_recovery_key 1 testuser

And here is my problem, this is the message I get:

**Store this link safely, as it will allow anyone to access authentik as testuser.
/recovery/use-token/EEEEEEEEEEEEAAAAAAAAAAAAAAAAAAAAAAAHHHHHHHHHHHHHH1111111111111111XXXXXXX/**

This is not a link. If i copy and paste this line to my authentik link, i get an error message that the page cannot be found:

https://my.authentik.tld/recovery/use-token/....
https://my.authentik.tld/if/flow/recovery/use-token/....
https://192.168.1.99:9000/recover/use-token/....and so on...

What am I missing? (sorry if it's a stupid question)

**Reply by scraejtp** [reply]
I can try it again on my installation later to check, but your steps look the same to me. The only thing worth noting is that I used an existing user, so I think in your case "testuser" needs to be preexisting to work.

I am a fledgling user myself, so if you continue to have issues you will get more attention starting your own post though.

**Reply by faustlod** [reply]
Thank you! Please let me know the outcome. I’d also like to point out that the user mentioned in the description has already been created. It was initially included as a placeholder under the name "testuser," but let’s stick with the "akadmin" user instead. I tested this on version 2024.10.1 as well, and the same issue persists.

**Reply by faustlod** [reply]
Any news?
It would be very helpful if somebody copies a valid link (with no sensitive data of course) for just to see, what should I see, when I'm generating a token.

**Reply by scraejtp** [reply]
Sorry for the delay, dropped off my radar. 

I tried the recovery login and it worked well for me locally or through my reverse proxy. The link it provides is a single login, will only work once for each link/token generated. I generated my login for an existing admin user, the default akadmin user.

When I tried to create a recovery key for a non-existing user I received an error, User 'xxx' not found.

Direct copy/paste for my local login example:
http://192.168.1.3:9000/recovery/use-token/9irBGsAfd6bXa7QxBjBeengMBo3I8myk6CtzVUxDyCz6fMlUubNNVg7NtV5N/

**Reply by faustlod** [reply]
Thank you very much! I had a feeling this might be what the full URL looks like. I also tried this format, but unfortunately, the application couldn't find the associated flow. :/
Among other things, my recovery flow is defined according to the default settings, so I haven't modified it, and it works perfectly. I'll wait a bit longer; it would be great if this issue could be resolved as well.
